.github/workflows/integration.yml

```diff
@@ -126,6 +126,15 @@ jobs:
         with:
           os: 'linux'
 
+      - name: Install Oracle ODPI-C
```

Comment:
Installation takes ~5 seconds, simplified version of 
https://github.com/kubo/rust-oracle/blob/master/.github/workflows/run-tests.yml#L49